nexus/src/internal_api/http_entrypoints.rs

```diff
@@ -170,6 +171,33 @@ async fn dataset_put(
     Ok(HttpResponseOk(DatasetPutResponse { reservation: None, quota: None }))
 }
 
+/// Report that a service within a pool has come online.
```

Comment:
Nit, "within a pool" is a bit misleading; service are generally decoupled from storage

Reply:
fixed in 0ca8a75d